lib/src/components/painter/chart_engine.dart

```diff
@@ -14,10 +14,10 @@ const double kLineStrokeWidth = 0.8;
 const double kBarWidthRatio = 0.7;
 const double kBarPaddingWidthRatio = (1 - kBarWidthRatio) / 2;
 
-const Color kLineColor1 = Color(0x44757575);
-const Color kLineColor2 = Color(0x77757575);
-const Color kLineColor3 = Color(0xAA757575);
-const Color kTextColor = Color(0xFF757575);
+ Color kLineColor1 = Color(0x44757575);
+ Color kLineColor2 = Color(0x77757575);
+ Color kLineColor3 = Color(0xAA757575);
+ Color kTextColor = Color(0xFF757575);
```

Comment:
Could you change here as const and then pass the style prop to the child component? I think it's not good using global mutable variables.